dlrs/main/classes/ManageLookupRollupSummariesController.cls

```diff
@@ -156,6 +175,52 @@ public with sharing class ManageLookupRollupSummariesController {
     return null;
   }
 
+  public void hasChildTriggers() {
+    try {
+      if (LookupRollupSummary != null) {
+        RollupSummary rs = new RollupSummary(LookupRollupSummary);
+        String childTrigger = RollupSummaries.makeTriggerName(rs);
+        String parentTrigger = RollupSummaries.makeParentTriggerName(rs);
+        ApexTriggersSelector selector = new ApexTriggersSelector();
+        Map<String, ApexTrigger> loadTriggers = selector.selectByName(
+          new Set<String>{ ChildTrigger, ParentTrigger }
+        );
+
+        switch on loadTriggers.size() {
+          when 2 {
+            Apexpages.addMessage(
+              new Apexpages.Message(
+                Apexpages.severity.CONFIRM,
+                'Rollup has child and parent triggers deployed. Can activate'
+              )
+            );
+          }
+          when 1 {
+            Apexpages.addMessage(
+              new Apexpages.Message(
+                Apexpages.severity.warning,
+                'Rollup has only one trigger deployed. Cannot activate'
+              )
+            );
+          }
+          when else {
+            Apexpages.addMessage(
+              new Apexpages.Message(
+                Apexpages.severity.fatal,
+                'Rollup does not have any triggers deployed. Cannot activate'
+              )
+            );
+          }
+        }
+      }
+    } catch (Exception e) {
+      //ultimately if there is an error it should not effect the rollup summary functionality. Probably better to fail silently.
```

Comment:
doesnt the comment contradict the code? we don't fail silently?

Reply:
Wanted to ask about this in dev meeting. Not sure if its bad practice to leave empty catch block.